apps/src/applab/applab.js

```diff
@@ -1040,7 +1041,11 @@ Applab.init = function(config) {
     }
     var designModeButton = document.getElementById('designModeButton');
     if (designModeButton) {
-      dom.addClickTouchEvent(designModeButton, Applab.onDesignModeButton);
+      // Simulate a run button click, to load the channel id.
+      var designModeClick = studioApp.runButtonClickWrapper.bind(
```

Comment:
This is following the same pattern as `viewDataClick` above, based on the code @davidsbailey wrote to ensure a `channel` is present before any action that requires a channel ID.

Dave, is the plan for this code to go away in the future?